frontend/src/components/dashboard/components/categories-list/categories-list.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from 'react';
+
+import { Category } from './components/category/category';
+import styles from './styles.module.scss';
+
+type Category = {
+  id: number;
+  name: string;
+};
+
+type Props = {
+  items: Category[];
+};
+
+const CategoriesList = ({ items }: Props): ReactElement => {
```

Comment:
```suggestion
const CategoriesList: FC<Props> = ({ items }) => {
```
I think it's better to create a component this way

Reply:
Why is it better way?

Reply:
FC is more explicit about the return type and provides some default props
But of course, I may be wrong, it can be not a big difference and a matter of preference
Let's see what coaches think about that